parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageReadStore.java

```diff
@@ -88,7 +89,8 @@ static final class ColumnChunkPageReader implements PageReader {
         byte[] fileAAD,
         int rowGroupOrdinal,
         int columnOrdinal,
-        ParquetReadOptions options) {
+        ParquetReadOptions options,
+        ParquetMetricsCallback callback) {
```

Comment:
Why not get the `callback` by the `options.getMetricsCallback`?

Reply:
Of course. My oversight.